repository/assets_maintenance_repo.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type AssetMaintenanceRepo interface {
-	InsertMaintenanceActivity(ctx context.Context, assets_id domain.UUID, req domain.MaintenanceActivity) (*domain.MaintenanceActivity, error)
+	InsertMaintenanceActivity(ctx context.Context, asset_id domain.UUID, req domain.MaintenanceActivity) (*domain.MaintenanceActivity, error)
```

Comment:
We are getting AssetId in domain.MaintenanceActivity obj. We can remove this extra parameter.

Reply:
Follow naming convention for variable names. e.g. assetId